docs/getting-started/how-prisma-schemas-are-converted-into-entities.md

```diff
@@ -0,0 +1,283 @@
+| Boolean | Boolean | Same Boolean type |
+| DateTime (@default(now())) | CreatedAt | DateTime with now() default converts to CreatedAt |
+| DateTime (@updatedAt) | UpdatedAt | DateTime with updatedAt attribute converts to UpdatedAt |
+| DateTime | DateTime | Plain DateTime remains DateTime |
+| Float, Decimal | Decimal | Float and Decimal convert to Decimal |
+| Int, BigInt | WholeNumber | Int and BigInt convert to WholeNumber |
+| String | SingleLineText | String converts to SingleLineText |    
+| Json | Json | Json types are equivalent |  
+| Enum | OptionSet | Enum converts to OptionSet |
+| Enum[] | MultiSelectOptionSet | Enum array converts to MultiSelectOptionSet |
+| Model relation | Lookup | Relation to another model becomes Lookup |
+
+## Log Warnings
+
+While your `schema.prisma` file is processing you'll see various types of logs like `(Info)` and `(Warning)` giving you information on what's happening in real-time.
+
+The schema will still process with warnings, but it's possible you might want to make some changes to your schema file and re-upload after correcting the warnings.
+
+### 1. Custom Attributes on enum keys
+
+Amplication currently does not support adding custom attributes to enum and enum keys.
+Instead, Amplication instead allows you to define options as an `optionSet` or `multiSelectOptionSet`.
+
+So if your Prisma schema includes `@map` or `@@map` custom attributes on an enum or enum keys, these attributes will not be accurately represented after processing.
+The final Prisma schema file will not contain these mapping attributes.  
+
+If your app logic relies on `@map` and `@@map` custom attributes, you'll need to manually adjust the generated `schema.prisma` file or your app code to ensure everything works correctly.
+
+Here's how that log might look:
+
+```bash title="Amplication Upload Log" 
+2023-07-06T09:47:39.410Z (Warning) The option 'PENDING' has been created in the enum 'OrderStatus', but its value has not been mapped
+2023-07-06T09:47:39.410Z (Info) The option 'COMPLETED' has been created in the enum 'OrderStatus'
+2023-07-06T09:47:39.410Z (Info) The option 'CANCELLED' has been created in the enum 'OrderStatus'
+2023-07-06T09:47:39.411Z (Warning) The option 'INDIVIDUAL' has been created in the enum 'CustomerType', but its value has not been mapped  
+2023-07-06T09:47:39.411Z (Warning) The option 'COMPANY' has been created in the enum 'CustomerType', but its value has not been mapped
+```
+
+### 2. Composite IDs on models   
+
+A model in Prisma can use the `@@id` custom attribute and combine fields to create a unique identifier for each record.
+But, due to current limitations, Amplication requires that each model have a single, distinct `id` field.   
+
+During schema processing. Amplication modifies IDs created with the `@@id` custom attribute to be `@@unique` custom attributes, which are supported.
+It also introduces a new `id` field of type `String` to each model.   
```

Comment:
@abrl91 is it string?

Reply:
@yuval-hazaz @abrl91 I was also referring to this line to confirm whether it is a `String` or `WholeNumber`?